### PR TITLE
Support added for GL-MC-001P

### PR DESCRIPTION
**Pull request by grostim**
"Pro" version of GL-MC-001

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -6920,6 +6920,14 @@ const devices = [
         extend: gledopto.light_onoff_brightness_colortemp_colorxy,
         meta: {disableDefaultResponse: true},
     },
+    {
+        zigbeeModel: ['GL-MC-001P'],
+        model: 'GL-MC-001P',
+        vendor: 'Gledopto',
+        description: 'Zigbee USB mini LED controller RGB + CCT PRO',
+        extend: gledopto.light_onoff_brightness_colortemp_colorxy,
+        meta: {disableDefaultResponse: true},
```
Comment (Koenkk) [inline devices.js:6929]: Is disableDefaultResponse really needed here? (can you try to remove it and check if everything works as expected)

**Comment by grostim** [comment]
It was activated on the non-pro version. I am not sure if i need it.
I will try tonight.

What should I expect if I remove it and it was needed ?

**Comment by Koenkk** [comment]
@grostim if it's needed, you will see time out errors in the log when sending commands.

Note to self: after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/473

**Comment by grostim** [comment]
Ok.  I have tested for a while. No visible issues without the meta key.

Le lun. 30 nov. 2020 à 19:07, Koen Kanters <notifications@github.com> a
écrit :

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**